### PR TITLE
[Revert](merge-on-write) Don't use delete bitmap to mark delete for rows with delete sign when sequence column doesn't exist 

### PR DESCRIPTION
**Pull request by bobhan1**
## Proposed changes

This PR reverts the optimization for delete sign in https://github.com/apache/doris/pull/24011

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by bobhan1** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Review by liaoxin01 — APPROVED** [review]
LGTM

File: regression-test/suites/unique_with_mow_p0/partial_update/test_partial_update_delete_sign.groovy [hunk 1]
```diff
@@ -55,69 +55,9 @@ suite('test_partial_update_delete_sign') {
     sql "set skip_storage_engine_merge=true;"
     sql "set skip_delete_bitmap=true;"
     sql "sync"
-    // skip_delete_bitmap=true, skip_delete_sign=true
-    qt_1 "select k1,c1,c2,c3,c4,__DORIS_DELETE_SIGN__ from ${tableName1} order by k1,c1,c2,c3,c4,__DORIS_DELETE_SIGN__;"
```
Comment (zhannngchen) [inline regression-test/suites/unique_with_mow_p0/partial_update/test_partial_update_delete_sign.groovy:59]: don't remove these cases

**Comment by bobhan1** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by bobhan1** [comment]
run pipelinex_p0

**Pull request merged**